### PR TITLE
put install command under util

### PR DESCRIPTION
**Pull request by joshmoore**
see: https://github.com/spacetx/starfish/pull/1026

### CODE REVIEW

File: starfish/starfish.py [hunk 1]
```diff
@@ -72,8 +72,18 @@ def version():
...
+def util():
+    """
+    house-keeping commands for the starfish library
+    """
+    pass
+
+@util.command()
 def install_strict_dependencies():
+    """
+    warning! updates different packages in your local installation
```
Comment (ttung) [inline starfish/starfish.py]: does this need to be in the help text?

Comment (joshmoore) [reply starfish/starfish.py]: The “warning”? Nah. But there aren’t many packages that install stuff themselves.

Comment (ttung) [reply starfish/starfish.py]: Or printed out to the console? Actually the help text for this command is pretty unhelpful.

Comment (joshmoore) [reply starfish/starfish.py]: Ha. Fair enough. :smile: But I'm misusing your sign-off to get this into your PR where we can discuss a help message better than `None` and "unhelpful"

**Pull request merged**